### ISSUE TITLE
Make the solution changing time independent of timezone

### ISSUE THREAD

**Issue by pavo-etc** [pavo-etc/minecraftle#12]
Currently the game only updates the solution at midnight in Perth Australia.  This is a hack.  Can fix this with another hack!

Make it so that the server generates a few days worth of puzzles, 2 days back 2 days forward. Then let the client pick which solution to use based on their local time.

Why the fuck doesn't js have a way to seed random(). Ugh

**Comment by pampeffyALT** [comment]
Yeah I make videos on this daily and was really annoyed that I had to wait until 11am each day, would be nice but not a priority.

**Comment by pavo-etc** [comment]
I think latest commit should fix this (d4b10e0), would love it if people in other timezones can let me know.

**Comment by TheVanadium** [comment]
In init.js line 364, the `generateSummary` function attempts to get the element with the id `timzone`. However, upon searching the repository, no such id exists, but an id `timezone` _does_ exist. Is this a typo?

**Comment by pavo-etc** [comment]
@TheVanadium ah yes this is a relic of my previous hack.  There is a timezone element in the main page HTML:

```html
<div id="solutions">
  <div date="2023-11-17" solution="stone_axe"></div>
  <div date="2023-11-18" solution="gold_block"></div>
  <div date="2023-11-19" solution="composter"></div>
</div>
 
<div id="timezone">Australia/Perth</div>

<div class="inv-background ingredients" id="ingredients">
  <div class="flex">
    <h2 class="left">Crafting Ingredients</h2>

```

The timezone is hardcoded in the backend to be Perth Australia is inserted into the #timezone element.  Since the puzzle reset used to be at midnight in Perth I made it so the summary function was always set to use that timezone for the summary date.

This is no longer needed since the puzzle resets at user local midnight.

**Comment by pavo-etc** [comment]
I guess this never properly worked because of the `timzone` typo.  Its not necessary now anyway

**Comment by pavo-etc** [comment]
Is resolved in #31 using seeded random JS library
